alembic/operations/ops.py

```diff
@@ -171,7 +171,7 @@ def from_constraint(cls, constraint: Constraint) -> DropConstraintOp:
             sqla_compat.constraint_name_or_none(constraint.name),
             constraint_table.name,
             schema=constraint_table.schema,
-            type_=types[constraint.__visit_name__],
+            type_=types.get(constraint.__visit_name__, "generic"),
```

Comment:
change "generic" to `None`, since we have another lookup for this when the operation actually runs

Reply:
Changed in latest commits.